jdd.js

```diff
@@ -1024,6 +1024,28 @@ var jdd = {
                     }
                 });
                 return true;
+            } else if (document.getElementById(id).value.trim().substring(0, 5).toLowerCase() === 'data:') {
+                var val = document.getElementById(id).value.trim();
+                if (val.length < 14 || val.substring(5,12) !== 'base64,') {
```

Comment:
I believe that your implementation will not correctly parse URLs that specify a mediatype. A mediatype is a MIME type that is optionally pretended (see the MDN docs you link to in `README.md`). In my opinion you should at least accept `application/json`, `text/plain`, and `application/octet-stream`. Or you may just ignore the mediatype, accept any value and treat the content as JSON anyway.

I'll check whether there's a standardized JavaScript API that could help you here. I am on my phone for the rest of the day (in UTC+01:00) and will do that tomorrow.

Reply:
I've made a code change to manually support the text and JSON mimetypes for now.  That's probably good enough if we can't find better library support.